--- a/qubes-rpc/Makefile
+++ b/qubes-rpc/Makefile
@@ -9,13 +9,13 @@
 		$(DESTDIR)/etc/qubes-rpc/policy/qubes.InputTablet
 
 install-vm:
-	install -d $(DESTDIR)/usr/lib/systemd/system
+	install -d $(DESTDIR)/lib/systemd/system
 	install -m 0644 \
 		qubes-input-sender-keyboard@.service \
 		qubes-input-sender-keyboard-mouse@.service \
 		qubes-input-sender-mouse@.service \
 		qubes-input-sender-tablet@.service \
-		$(DESTDIR)/usr/lib/systemd/system
+		$(DESTDIR)/lib/systemd/system
 	install -d $(DESTDIR)/lib/udev/rules.d
 	install -m 0644 qubes-input-proxy.rules \
 		$(DESTDIR)/lib/udev/rules.d/90-qubes-input-proxy.rules
